core/dbt/task/rpc/project_commands.py

```diff
@@ -305,3 +307,22 @@ def output_results(results):
             output=[json.loads(x) for x in results],
             logs=None
         )
+
+
+class RemoteBuildProjectTask(RPCCommandTask[RPCBuildParameters], BuildTask):
+    METHOD_NAME = 'build'
+
+    def set_args(self, params: RPCRunParameters) -> None:
```

Comment:
Should this be `RPCBuildParameters`?

Reply:
I committed this fix locally 6 days ago... forgot to push... classic